animation.pug

```diff
@@ -7,4 +7,8 @@ block prepend title
     = "Animation - "
 
 block content
-    p Page to come.
+    p.
+        Many fan created animations have been created to display in the same way the official games do it! The animation editor is available here: 
+        <a href="https://gamebanana.com/tools/6426"></a>. Load in an "animation.xml" file, select sprites to use, path out animations, and preview them!
```

Comment:
This is fine for now, but in the future we should move the download here and add equivalent help information here, on our official source.